intl/en/messages.json

```diff
@@ -0,0 +1,10 @@
+{
+  "1092fa6c86f124928e0f4d9d4711ebc9": "options must be an object: {0}",
+  "66a6a44b45094ef45fdcb0201a26fc6d": "table is a required string argument: {0}",
+  "b7c60421de706ca1e050f2a86953745e": "No arguments - could not create {{Enum}}.",
+  "80a32e80cbed65eba2103201a7c94710": "Model not found: {0}",
+  "610db3e4d1e877d985000ed09fb01fb9": "callback should be a function",
+  "a354e762822e356af4bfa3299fd29b0a": "{{MySQL}} regex syntax does not respect the {{`g`}} flag",
+  "cecffcfbf707d97548dcd5859c80b171": "{{MySQL}} regex syntax does not respect the {{`i`}} flag",
+  "d230064cbc14fe9bfcf2fb2072c16fb2": "{{MySQL}} regex syntax does not respect the {{`m`}} flag"
```

Comment:
you forgot to to do `slt-globalize -e` and `slt-globalize -l` to update this file?